lyrics.py

```diff
@@ -0,0 +1,97 @@
+# but would likely have to rework dictionary entries and exceptional phrases at intro and finish
+things_she_ate = ["fly", "spider", "bird", "cat", "dog", "goat", "cow", "horse"]
+
+# dictionary for second line of each phrase
+dict_second_line = {
+    "spider" : "It wriggled and jiggled and tickled inside her.",
+    "bird" : "How absurd to swallow a bird.",
+    "cat" : "Imagine that, to swallow a cat!",
+    "dog" : "What a hog, to swallow a dog!",
+    "goat" : "Just opened her throat and swallowed a goat!",
+    "cow" : "I dont know how she swallowed a cow!"
+}
+
+# dictionary for repeated verses
+dict_verse = {
+    "fly" : "I dont know why she swallowed the fly. Perhaps she'll die.",
+    "spider" : "She swallowed the spider to catch the fly.",
+    "bird" : "She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.",
+    "cat" : "She swallowed the cat to catch the bird.", 
+    "dog" : "She swallowed the dog to catch the cat.",
+    "goat" : "She swallowed the goat to catch the dog.",
+    "cow" : "She swallowed the cow to catch the goat.",
+    "horse" : "I know an old lady who swallowed a horse. She's dead of course!"
+}
+# define empty list that repeating verses will be added to the front of
+verse = []
+
+##############################################################################
+
+# print first line of each phrase     ("string")
+def first_line(animal):
+    print("I know an old lady who swallowed a " + animal + ".")
+
+#print second line of each phrase     (use "key" to call entry from dict_second_line)
+def second_line(animal):
+    print(dict_second_line[animal])
+
+# print  list of accumulated verses and a blank line
+def assemble(listed_verses):
+    for animal in listed_verses:
+        print(dict_verse[animal])
+    print("")
+
+# adds new animal verse to beginning of repeating animal verse list      new empty list defined  in section with dictionaries
+def add_repeating(animal):
+    verse.insert(0,animal)
+
+#################################
+
+# prints three blank lines so poem isnt squished up against other stuff in terminal
+def setup():
+    print("")
+    print("")
+    print("")
+
+# adds first animal to front of empty new list and prints it     
+# this phrase gets special treatment because first is different
+def intro(animal):
+    add_repeating(animal)
+    assemble(verse)
+
+# prints phrases that follow a repeatable pattern    call with  "string" that is same as corresponding dictionary keys
+# first line, print second line, adds relevant line to repeating verse list, print repeating verses list
+def stanza(animal):
+    first_line(animal)
+    second_line(animal)
+    add_repeating(animal)
+    assemble(verse)
+
+# prints last phrase     because its different      call with key from dict_verse
+def finish(animal):
+    print(dict_verse[animal])
+
+##########################
+# arranges poem elements together in order
+def recite_poem():
```

Comment:
This code is all really sensibly organized and broken down. Great job! Can you think here about how you could use a loop to avoid repetition?